### DISCUSSION TITLE
Add the function of disable the client by the client Id

### DISCUSSION THREAD

**Discussion by mthgh0818** [rabbitmq/rabbitmq-server#9224] [Ideas]
is there a way to prohibit some client Id on the webUI, after disabling, the device can no longer connect with this clientId

**Comment by michaelklishin** [comment]
MQTT client IDs are not used for access control.

I'd need to check if client ID is passed on to [custom authN backends](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_http). If that is/was the case, you would be able to
blacklist clients by client ID in your own HTTP(S) service that acts as an authN plugin.

**Reply by mthgh0818** [reply]
dear michaelklishin,I tested authN backends authentication method and did not get the clientId，Is there  a way Obtain the Client ID by authN？

**Reply by michaelklishin** [reply]
I cannot suggest much since you haven't shared any details at all, even RabbitMQ version used. We do not guess in this community.

https://github.com/rabbitmq/rabbitmq-server/discussions/10192 sounds relevant (for 3.12.x)
